marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/auth/WebSecurityConfiguration.java

```diff
@@ -71,7 +71,6 @@ protected void configure(HttpSecurity http) throws Exception {
      */
     protected String[] getAlwaysPermittedPatterns() {
         return new String[]{
-            "/api/environment/initialized",
             "/websocket/**",
```

Comment:
/websocket/** can be removed too, right?

Reply:
yeah, I have removed it